corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -676,6 +663,21 @@ def get_identifier_source_id(requests, identifier_type):
         raise OpenmrsHtmlUiChanged(
             'Domain "{}": Unexpected OpenMRS login page at "{}".'.format(requests.domain_name, response.url)
         )
+    return session
```

Comment:
this return value doesn't get used. it appears to be modifying the session in place